mate-panel/panel.c

```diff
@@ -657,8 +657,8 @@ drop_urilist (PanelWidget *panel,
 			can_exec = g_file_info_get_attribute_boolean (info,
 								      G_FILE_ATTRIBUTE_ACCESS_CAN_EXECUTE);
 
-			if (mime &&
-			    g_str_has_prefix (mime, "image")) {
+			if (uris[1] == NULL /* if there's only one URI */ &&
```

Comment:
It's better to use [g_strv_length](https://developer.gnome.org/glib/stable/glib-String-Utility-Functions.html#g-strv-length) instead of `uris[1] == NULL` for a NULL terminated string array. (readability)

Reply:
OK, changed to use it.  We could then user the length in the loop, but that's kinda out of scope.